egs/ami/s5/local/nnet3/run_lstm.sh

```diff
@@ -79,28 +81,33 @@ if [ $stage -le 7 ]; then
      /export/b0{3,4,5,6}/$USER/kaldi-data/egs/ami-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
   fi
 
-  steps/nnet3/lstm/train.sh --stage $train_stage \
-    --num-epochs 3 --num-jobs-initial 2 --num-jobs-final 12 \
-    --egs-dir "$common_egs_dir" \
+
+  #--online-ivector-dir exp/$mic/nnet3/ivectors_train_hires_sp2 \
+
+  steps/nnet3/lstm/train_delay.sh --stage $train_stage \
+    --label-delay $label_delay \
+    --num-epochs $num_epochs --num-jobs-initial 2 --num-jobs-final 12 \
     --num-chunk-per-minibatch $num_chunk_per_minibatch \
     --samples-per-iter $samples_per_iter \
     --splice-indexes "$splice_indexes" \
     --feat-type raw \
     --online-ivector-dir exp/$mic/nnet3/ivectors_${train_set}_hires \
     --cmvn-opts "--norm-means=false --norm-vars=false" \
+    --io-opts "-tc 12" \
```

Comment:
You can make these changes a separate pull request.
But don't you think a 3-layer LSTM is a more reasonable default?
Dan


On Mon, Sep 14, 2015 at 5:51 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/ami/s5/local/nnet3/run_lstm.sh
> <https://github.com/kaldi-asr/kaldi/pull/136#discussion_r39452143>:
>
> >      --num-chunk-per-minibatch $num_chunk_per_minibatch \
> >      --samples-per-iter $samples_per_iter \
> >      --splice-indexes "$splice_indexes" \
> >      --feat-type raw \
> >      --online-ivector-dir exp/$mic/nnet3/ivectors_${train_set}_hires \
> >      --cmvn-opts "--norm-means=false --norm-vars=false" \
> > +    --io-opts "-tc 12" \
>
> io-opts is not longer necessary for the new get_egs.sh script, you can
> remove that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/136/files#r39452143>.
>